src/index.js

```diff
@@ -1,84 +1,70 @@
-import {
-  generateColorExtension,
-  cgColorString,
-  generateFontExtension,
-  options,
-  linearGradientLayer,
-  radialGradientLayer,
-} from './helpers';
+import Zepcode from './zepcode';
 
 function styleguideColors(context, colors) {
-  return generateColorExtension(colors, options(context));
+  return new Zepcode(context).generateColorExtension(colors);
 }
```

Comment:
Why did you use instance method instead of static? Sorry if I use the wrong terminology

Reply:
Because we need shared context (instance internal property): https://github.com/artemnovichkov/zepcode/blob/e376ce11b6d117e66fe61989c81b41a636edc0b1/src/templates/color-extension.js#L14